### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by jgrayson676**
I used `twitter.handle` and `twitter.id` as temporary names for those fields until I hear back what the actual names should be.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# leadconduit-integration-twitter
\ No newline at end of file
+# LeadConduit Twitter Integration
+
+This module is for use on the LeadConduit platform with Twitter Lead Generation Cards. Please see the [license agreement](http://creativecommons.org/licenses/by-nc-nd/4.0/).
```
Comment (cgrayson) [inline README.md]: It would be nice to have this link to a Twitter page about Lead Gen Cards.

File: src/lead_card.coffee [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+#
+
+supportedMimeTypes = [
+  'application/x-www-form-urlencoded'
+]
+
+selectMimeType = (contentType) ->
+  contentType = contentType or 'text/xml'
+  contentType = 'text/xml' if contentType is '*/*'
+  mimeparse.bestMatch(supportedMimeTypes, contentType)
```
Comment (cgrayson) [inline src/lead_card.coffee]: Wonder if this array & function are worthwhile, since there's only one supported type.

**Comment by cgrayson** [comment]
Otherwise LGTM

File: src/lead_card.coffee [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    throw new HttpError('415', { 'Content-Type': 'text/plain' }, 'Content-Type header is required')
+
+  # ensure valid mime type
+  mimeType = mimeparse.bestMatch(['application/x-www-form-urlencoded'], contentType)
+  unless mimeType is 'application/x-www-form-urlencoded'
+    throw new HttpError('406', { 'Content-Type': 'text/plain' }, 'MIME type in Content-Type header is not supported. Use only application/x-www-form-urlencoded')
+
+  lead = querystring.parse(req.body)
+
+  firstSpace = lead.name?.indexOf ' '
```
Comment (cgrayson) [inline src/lead_card.coffee]: ICYMI - there's now a [better way to do this](https://github.com/activeprospect/leadconduit-integration-facebook/pull/14/files). :sunglasses:

Comment (jgrayson676) [inline src/lead_card.coffee]: Will you (or should I) add support for middle name and salutation now that those are standard fields?

Comment (cgrayson) [inline src/lead_card.coffee]: Good point - I'll do that.

**Pull request merged**